src/site/resources/.htaccess

```diff
@@ -32,12 +32,14 @@ RewriteRule "^(.+)$" "$1.html"
 # From this point rules match against the URI to the real file, with "/log4j/2.x/" stripped.
 
 # The content moved between pages
+RewriteRule "^articles\.html$" "manual/index.html" [R=permanent]
 RewriteRule "^log4j-core/apidocs(.*)$" "javadoc/log4j-core$1" [R=permanent]
 RewriteRule "^manual/api-separation\.html$" "manual/api.html" [R=permanent]
-RewriteRule "^manual/scala-api\.html$" "/log4j/scala/index.html" [R=permanent]
+RewriteRule "^manual/scala-api\.html$" "/log4j/scala" [R=permanent]
```

Comment:
```suggestion
RewriteRule "^manual/scala-api\.html$" "/log4j/scala/latest/index.html" [R=permanent]
```

Reply:
Sorry, I am against this. `/log4j/scala` is an external site and I don't want to have assumptions on its implementation. This approach is future-proof against the problems you're suggesting to fix, i.e., `/index.html` has been moved to `/latest/index.html`.